application/src/main/java/org/togetherjava/tjbot/features/chaptgpt/AIResponseParser.java

```diff
@@ -0,0 +1,88 @@
+package org.togetherjava.tjbot.features.chaptgpt;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+/**
+ * Represents a class to break up long text blocks into smaller blocks which work with Discord's
+ * API. Initially constructed to break apart text from AI text generation APIs.
+ */
+public class AIResponseParser {
+
```

Comment:
remove this line maybe?

Reply:
Which line number are you talking about?

Reply:
16 the empty line